WooCommerce/WooCommerce.xcodeproj/xcshareddata/xcschemes/WooCommerce.xcscheme

```diff
@@ -129,7 +129,7 @@
          </CommandLineArgument>
          <CommandLineArgument
             argument = "-simulate-stripe-card-reader"
-            isEnabled = "NO">
+            isEnabled = "YES">
```

Comment:
We might want to revert this 😄 

Fixed in https://github.com/woocommerce/woocommerce-ios/pull/13040/commits/c5d88618b1040dd667be5d3701b0b8f8cfd21b31